### PR TITLE
Improve the README.md

### PR DESCRIPTION
**Pull request by afrittoli**
Add a link to CONTRIBUTING.md and improve the README.md to include links to
the various documents.

Remove the code of conduct that is now hosted in the .github repo.

Signed-off-by: Andrea Frittoli <andrea.frittoli@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,44 @@
-# CDEvents
+# ![CDEvents](cdevents_horizontal-color.png)
 
-We are working on a draft version of our specification that is not yet ready for implementation. For more information check out our [spec](./spec.md)
+CDEvents is a common specification for Continuous Delivery events, enabling
+interoperability in the complete software production ecosystem.
+
+It's an incubated project at the
```
Comment (oleg-nenashev) [inline README.md:6]:
```suggestion
It's an incubating project at the
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,44 @@
...
+* [Go](https://github.com/cdevents/sdk-go)
+
+## Community
+
+### How to get involved
+
+[Reach out](governance.md#project-communication-channels) to see what we're up
+via:
+
+* [slack](https://cdeliveryfdn.slack.com/archives/C030SKZ0F4K)
```
Comment (oleg-nenashev) [inline README.md:31]:
```suggestion
* [slack](https://cdeliveryfdn.slack.com/archives/C030SKZ0F4K) - [Join the Slack Channel](https://join.slack.com/t/cdeliveryfdn/shared_invite/zt-nwc0jjd0-G65oEpv5ynFfPD5oOX5Ogg)
```

**Review by oleg-nenashev — COMMENTED** [review]
joining CDF Slack is not trivial - a link is needed

**Comment by MarckK** [comment]
Thank you for adding these docs, @afrittoli!  🎉

**Comment by afrittoli** [comment]
> joining CDF Slack is not trivial - a link is needed

Good point :) Thanks for the review!

File: contributing.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Contributing to CDEvents
+
+Thank you for contributing your time and expertise to CDEvents. This document describes the contribution guidelines for the project.
+
+**Note:** Before you start contributing, you must read and abide by our **[Code of Conduct](./code-of-conduct.md)**.
```
Comment (m-linner-ericsson) [inline contributing.md:5]: Just realized that both the contributing and code of conduct probably should be in the .github repo. See https://github.com/eiffel-community/.github for an example

**Review by m-linner-ericsson — COMMENTED** [review]
Good work but maybe we should move the contributing and code of conduct to the .github repo

File: README.md [hunk 4]
```diff
@@ -1,3 +1,44 @@
...
+
+The specification is currently draft and not ready for implementation.
+We are working towards our
+[first release v0.1](https://github.com/orgs/cdevents/projects/1).
+
+## CDEvents SDKs
+
+CDEvents is developing as set of SDKs:
+
+* [Go](https://github.com/cdevents/sdk-go)
```
Comment (m-linner-ericsson) [inline README.md:22]: @erkist Do you want Python to be listed here also?

File: contributing.md (hunk 3 above)
Comment (m-linner-ericsson) [reply contributing.md:5]: I would start with having it in the `.github` repo and then if needed we can think about duplicating them.

**Review by m-linner-ericsson — APPROVED** [review]

**Pull request merged**